### PR TITLE
chore: Fix rack vulnerabilities

### PR DESCRIPTION
**Pull request by etceterum**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -120,6 +120,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-20
+  x86_64-darwin-21
```
Comment (dknox20) [inline Gemfile.lock:123]: we might want to add ruby here: 

See: https://bundler.io/man/gemfile.5.html#PLATFORMS

**Review by dknox20 — DISMISSED** [review]
LGTM

File: Gemfile.lock (hunk 1 above)
Comment (etceterum) [reply Gemfile.lock]: I've checked `ue` and `buyer-intent-api` and both just have `ruby` in there. I suppose there are no reasons not to do the same so replacing

**Review by dknox20 — APPROVED** [review]
LGTM

**Pull request merged**